src/dynamic_sharing_service/policy/dynamic.policy.builder.js

```diff
@@ -122,6 +127,8 @@ module.exports = class DynamicPolicyBuilder {
   withSelfieAuthentication(enabled = true) {
     if (enabled) {
       return this.withWantedAuthType(SELFIE_AUTH_TYPE);
+    } else if (!enabled) {
```

Comment:
I think we can make this more concise, since it's immediately obvious this is the path taken if `!enabled`. This will flag up on [SonarLint](https://www.sonarlint.org/) as well once we have that enabled:
```suggestion
    } else {
```

Reply:
Eslint was complaining about this just being an "else" statement.  We might be able to remove the else completely, as due to the default parameter I think we can assume that if the original condition doesn't pass, we can attempt to remove the element from the wanted auth types.

Reply:
[This commit](https://github.com/getyoti/yoti-node-sdk/commit/067bdfe0f960725d452e43c112f4adbf47f3c8c1) should fix this.  Tests are still passing

Reply:
Looks good 👍